src/stream/src/executor/backfill/utils.rs

```diff
@@ -184,7 +184,7 @@ fn mark_chunk_inner(
 pub(crate) fn mapping_chunk(chunk: StreamChunk, output_indices: &[usize]) -> StreamChunk {
     let (ops, columns, visibility) = chunk.into_inner();
     let mapped_columns = output_indices.iter().map(|&i| columns[i].clone()).collect();
-    StreamChunk::new(ops, mapped_columns, visibility)
+    StreamChunk::new(ops, mapped_columns, visibility.into_visibility())
```

Comment:
Can we have `Into` here?

Reply:
I plan to refactor it into the following 2 methods:
```rust
StreamChunk::new(
    ops: impl Into<Arc<[Op]>>,
    columns: Vec<ArrayRef>,
)
StreamChunk::with_visibility(
    ops: impl Into<Arc<[Op]>>,
    columns: Vec<ArrayRef>,
    visibility: impl Into<Vis>,
)
```
and eventually get rid of `Option<Bitmap>` and only use `Vis` for visibilities.

But this would cause a big change to the codebase. So maybe next PR for sure?